src/csharp/Extensions/Microsoft.Spark.Extensions.Delta/Microsoft.Spark.Extensions.Delta.csproj

```diff
@@ -2,6 +2,7 @@
 
   <PropertyGroup>
     <TargetFramework>netstandard2.0</TargetFramework>
+    <IsPackable>true</IsPackable>
   </PropertyGroup>
```

Comment:
Can you add more fields like you found here: https://github.com/dotnet/spark/blob/master/src/csharp/Microsoft.Spark.Experimental/Microsoft.Spark.Experimental.csproj?

Reply:
Added the rest of the fields.